.github/workflows/test-package.yml

```diff
@@ -0,0 +1,34 @@
+name: test-sdk
+
+on: [push, pull_request]
```

Comment:
This should probably be constrained to the main branch, and add `workflow_dispatch:` so that the job can be re-run from the Actions console.

```yaml
on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
  workflow_dispatch:
```

Reply:
Agreed, @teolemon could you make the change?